CrowdState cur_state
float32 dt
---
CrowdState next_state
